biogrid/apps/biogrid-mvp/src/app/pages/simulate-page/simulate-page.tsx

```diff
@@ -3,42 +3,42 @@ import './simulate-page.css';
 import { Client } from '../../client';
 import { BiogridSimulationResults } from '../../build';
 
+/*
 enum SimulationStatus {
   NOT_STARTED,
   RUNNING,
   RAN,
 }
+*/
 
 export const SimulatePage = () => {
   const [simulationResults, setSimulationResults] = useState<
     BiogridSimulationResults
   >();
+  /*
   const [simulationStatus, setSimulationStatus] = useState(
     SimulationStatus.NOT_STARTED
   );
-
+  */
   const client = Client.getInstance();
+  /*
   async function runSimulation() {
     setSimulationStatus(SimulationStatus.RUNNING);
     await client.api.runBiogridSimulation();
     setSimulationStatus(SimulationStatus.RAN);
   }
+  */
   async function getSimulationResults() {
+    //setSimulationStatus(SimulationStatus.RUNNING);
+    await client.api.runBiogridSimulation();
+    //setSimulationStatus(SimulationStatus.RAN);
     const results = await client.api.getBiogridSimulationResults();
     setSimulationResults(results);
   }
 
   return (
     <div className="simulation">
-      <button onClick={() => runSimulation()}>Run the simulation</button>
-      {simulationStatus === SimulationStatus.RUNNING && (
-        <div>Loading your results...</div>
-      )}
-      {simulationStatus === SimulationStatus.RAN && (
-        <button onClick={() => getSimulationResults()}>
-          Get simulation results
-        </button>
-      )}
+      <button onClick={() => getSimulationResults()}>Get simulation results</button>
```

Comment:
what does the get simulation results button do on this page? users already put in their inputs and got results, so clicking this would simply rerun the simulation with the same variables? 
maybe instead, we should remove this and have a button at the bottom allowing users to "change inputs" which takes them back to the home page?

Reply:
clicking the button actually allows the user to see the results

Reply:
I think it would be good to lay out the user flow here, and keep in mind the UX principles of removing unnecessary clicks in order to make the UI as simple and easy to follow as possible.  So if the user flow is: (1) enter inputs, (2) get results, (3) re-enter inputs for new results I think the pages should be:
first page: enter inputs, click "run simulation" -> this click takes you to the second page with the results already loaded
second page: shows results, has a "re-run simulation with new inputs" button -> clicking this takes you back to the first page 

Happy to chat about it over meets :) let me know if that makes sense!

Reply:
sounds good. I will try to implement that